<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-J4AIRBB" failures="0" tests="2" name="org.shoppingmart.createcategory.CreateCategoryAndValidateInSubCategoryTest" time="0.001" errors="1" timestamp="2022-07-27T15:55:01 IST" skipped="1">
  <testcase classname="org.shoppingmart.createcategory.CreateCategoryAndValidateInSubCategoryTest" name="classSetup" time="0.001">
    <error message="Cannot invoke &quot;org.tyss.genericUtilty.WebDriverUtility.setupDriver(String)&quot; because &quot;this.webDriverUtility&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.tyss.genericUtilty.WebDriverUtility.setupDriver(String)" because "this.webDriverUtility" is null
at org.tyss.genericUtilty.BaseClass.classSetup(BaseClass.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </error>
  </testcase> <!-- classSetup -->
  <system-out/>
  <testcase classname="org.shoppingmart.createcategory.CreateCategoryAndValidateInSubCategoryTest" name="createCategoryAndValidateInSubCategory" time="0.000">
    <skipped/>
  </testcase> <!-- createCategoryAndValidateInSubCategory -->
  <system-out/>
</testsuite> <!-- org.shoppingmart.createcategory.CreateCategoryAndValidateInSubCategoryTest -->
